src/lib/variants.test.ts

```diff
@@ -169,6 +170,44 @@ describe('variant filters', () => {
         expect(got2).toBe(false);
     });
 
+    it('should filter by user location', () => {
+        // Test 1 - should return true if no location set in the test
+        const got1 = matchesCountryGroups.test(testDefault, targetingDefault);
```

Comment:
Even though the `testDefault` and `targetingDefault` already have the null/empty values we need here, what do you think about overriding explicitly here so it's really clear what's going on? For example:

```js
const test1 = {
  ...testDefault,
  locations: [],
};
```

Reply:
I didn't want to be adding unnecessary noise but seeing as this is a test it's probably worth doing for the sake of readability. Updated.